DQMOffline/RecoB/src/EffPurFromHistos.cc

```diff
@@ -602,4 +604,6 @@ void EffPurFromHistos::compute(DQMStore::IBooker& ibook) {
   }
 }
 
+#include <memory>
```

Comment:
@fwyzard - it kind of looks random here. Shouldn't it be at the beginning of the file? 
@smuzaffar - it's odd that the `unique_ptr` usage was allowed without its include. Do we run include checker?